from_cpython/Include/object.h

```diff
@@ -826,7 +826,7 @@ PyAPI_FUNC(void) _Py_AddToAllObjects(PyObject *, int force) PYSTON_NOEXCEPT;
 
 
 #define Py_INCREF(op) ((void)(op))
-#define Py_DECREF(op) ((void)(op))
+#define Py_DECREF(op) asm volatile("" : : "x"(op))
```

Comment:
It looks like this should be a `X` (capital x) constraint; lower case `x` apparently means "SSE register" :P